bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/service/BusinessPartnerService.kt

```diff
@@ -272,4 +280,51 @@ class BusinessPartnerService(
 
     }
 
+    @Scheduled(cron = "\${cleaningService.pollingCron:-}", zone = "UTC")
+    fun checkBpnUpdates() {
+
+        val poolPaginationRequest = PaginationRequest(0, 100)
+
+        val syncRecord = syncRecordService.getOrCreateRecord(SyncType.POOL_TO_GATE_OUTPUT)
+
+        val changelogSearchRequest = ChangelogSearchRequest(syncRecord.finishedAt)
+
+
+        val poolChangelogEntries = poolClient.changelogs.getChangelogEntries(changelogSearchRequest, poolPaginationRequest)
+        val poolUpdatedEntries = poolChangelogEntries.content.filter { it.changelogType.name == PoolChangeLogType.UPDATE.name }
+
+
+        val bpnA =
+            poolUpdatedEntries.filter { it.businessPartnerType == BusinessPartnerType.ADDRESS && it.changelogType == PoolChangeLogType.UPDATE }.map { it.bpn }
+        val bpnL = poolUpdatedEntries.filter { it.businessPartnerType == BusinessPartnerType.LEGAL_ENTITY && it.changelogType == PoolChangeLogType.UPDATE }
+            .map { it.bpn }
+        val bpnS =
+            poolUpdatedEntries.filter { it.businessPartnerType == BusinessPartnerType.SITE && it.changelogType == PoolChangeLogType.UPDATE }.map { it.bpn }
```

Comment:
Didn't you already filter for ChangelogType UPDATE? Why do you check for the UPDATE type here again?